src/NuGetForUnity/Editor/Configuration/NugetConfigFile.cs

```diff
@@ -402,6 +401,14 @@ public void Save([NotNull] string filePath)
                 config.Add(addElement);
             }
 
+            if (SlimRestore)
+            {
+                addElement = new XElement("add");
+                addElement.Add(new XAttribute("key", "slimRestore"));
+                addElement.Add(new XAttribute("value", Verbose.ToString().ToLowerInvariant()));
+                config.Add(addElement);
+            }
```

Comment:
```suggestion
            addElement = new XElement("add");
            addElement.Add(new XAttribute("key", "slimRestore"));
            addElement.Add(new XAttribute("value", SlimRestore.ToString().ToLowerInvariant()));
            config.Add(addElement);
```
Especially if we set `SlimRestore` to `true` by default I would always include the setting in the `NuGet.config` file.
Also there is a copy-paste error `Verbose` -> `SlimRestore`